sormas-backend/src/test/java/de/symeda/sormas/backend/externalemail/ExternalEmailFacadeEjbTest.java

```diff
@@ -650,6 +637,222 @@ public void testSendBulkEmailToCasePerson() throws MessagingException, IOExcepti
 		assertThat(relatedEntityUuids, hasItem(caze2.getUuid()));
 	}
 
+	@Test
+	public void testSendBulkEmailToCasePersonAttachmentsAndTemplateDocument() throws MessagingException, IOException {
+
+		CaseDataDto caze1 = creator.createCase(userDto.toReference(), personDto.toReference(), rdcf);
+
+		loginWith(userDto);
+
+		locationDto2 = LocationDto.build();
+		locationDto2.setStreet("Nauwieserstraße");
+		locationDto2.setHouseNumber("105");
+		locationDto2.setCity("Saarbrücken");
+		locationDto2.setPostalCode("66111");
+
+		PersonDto personDto2 = PersonDto.build();
+		personDto2.setFirstName("John");
+		personDto2.setLastName("Doe");
+		personDto2.setSex(Sex.UNKNOWN);
+		personDto2.setBirthdateYYYY(1980);
+		personDto2.setBirthdateMM(10);
+		personDto2.setBirthdateDD(22);
+		personDto2.setAddress(locationDto2);
+		personDto2.setPhone("+49 123 4567");
+		personDto2.setNationalHealthId("1987080412582");
+		personDto2.setEmailAddress("testEmail2@email.com");
+		getPersonFacade().save(personDto2);
+		CaseDataDto caze2 = creator.createCase(userDto.toReference(), personDto2.toReference(), rdcf);
+
+		Mockito.doAnswer(invocation -> {
+			assertThat(invocation.getArgument(0), is("testEmail@email.com"));
+			assertThat(invocation.getArgument(1), is("Email subject in template"));
+
+			StringWriter writer = new StringWriter();
+			IOUtils.copy(getClass().getResourceAsStream("/docgeneration/emailTemplates/cases/CaseEmail.cmp"), writer, "UTF-8");
+			String expectedContent = updateLineSeparatorsBasedOnOS(writer.toString());
+
+			assertThat(invocation.getArgument(2), is(expectedContent));
+			assertThat(invocation.getArgument(3), aMapWithSize(2));
+
+			return null;
+		}).doAnswer(invocation -> {
```

Comment:
on all bulk tests there are 2 invocations because there are 2 calls of emailService.sendEmail() method for each of the two cases selected for bulk email. Each call has different arguments so the asserts should be different. Actually, in the tests I added the important check is the assert of last argument which in case of this test has to contain 2 attchments (one attachment and one personalized file)